src/flows/recommendation_api/topics_flow.py

```diff
@@ -0,0 +1,59 @@
+from prefect import Flow, Parameter, unmapped, task
+from prefect.executors import LocalDaskExecutor
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    create_corpus_candidate_set_record,
+    load_feature_record,
+    feature_group,
+    validate_corpus_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+
+FLOW_NAME = get_flow_name(__file__)
+
+# Export approved corpus items by language and recency
+EXPORT_CORPUS_ITEMS_SQL = """
+SELECT 
+    approved_corpus_item_external_id as ID, 
+    topic
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS"
+WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', -90, current_timestamp())
+AND TOPIC = %(CORPUS_TOPIC_ID)s
+AND LANGUAGE = 'EN'
+"""
+
+GET_TOPICS_SQL = """
+SELECT 
+    curated_corpus_candidate_set_id, 
+    corpus_topic_id
```

Comment:
Optional suggestion: On line 35 we are relying on Snowflake turning these columns from lowercase to uppercase. We can remove this assumption by using double quotes to make our column names case-sensitive:
```suggestion
    "CURATED_CORPUS_CANDIDATE_SET_ID", 
    "CORPUS_TOPIC_ID"
```

Reply:
I have updated the sql to alias the column names with the explicitly capitalized "CAP" field names.

Also, updated the return task object names to uniquely identify the tasks and their relationships in the flow